frontend/src/js/components/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
```

Comment:
```suggestion
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```
So hab ich das gemacht. Wahrscheinlich hast du das später auch schon so ergänzt.

Reply:
Hatte ich tatsächlich noch nicht, ich hab es aber jetzt mal auf dem navigation-Branch hinzugefügt.